sdk/__init__.py

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from sdk.batch import Batch
+import time
```

Comment:
unused import ?

Reply:
It is actually, when we wait until each try to pull the results.